docs/contributing.rst

```diff
@@ -101,3 +126,23 @@ address, indicating that you agree to the `Developer Certificate of Origin
 	    this project or the open source license(s) involved.
 
 Use ``git commit -s`` to add the Signed-off-by tag.
+
+
+Releasing
+---------
+
+When cutting a new release, follow these steps:
+
+* update the version in ``fedora_messaging/__init__.py``
+* update the version in ``docs/conf.py``
```

Comment:
I have some fanciness to pull the version in the docs from __init__.py, I'll push a commit to this PR

Reply:
That that looks cool. So this line isn't necessary anymore.